### PR TITLE
[SS-04] build: add docker compose for local db set up

### PR DESCRIPTION
**Pull request by ksreyr**
## Changes Made 🎉

- [x] build: add docker compose for local db set up)[build: add docker compose for local db set up

### Describe Changes
 **Readme**
- updated to show how to use it

**Docker Compose**
- profiles added

**package.json**
- task to trigger the db setting up locally

**.env.template**
- updated with the docker url 

**.gitignore**
- updated to exclude IntelliJ idea IDE

## Related Issue(s)

- #4 

## Checklist ✅

- [x] My code follows the code style of this project.

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,12 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
+postgres_data
 # dependencies
 node_modules
 .pnp
 .pnp.js
 
+# webstrom
```
Comment (serudda) [inline .gitignore]:
```suggestion
# webstorm
```

File: README.md [hunk 2]
```diff
@@ -29,6 +29,18 @@ Here are the steps to execute the project:
    ```js
    DISCORD_CLIENT_ID= 'ENTER YOUR CLIENT ID';
    ```
+
+   4.1 (Optional) Generate local Database
```
Comment (serudda) [inline README.md]: Parece obvio, pero seria bueno indicar que es necesario instalar Docker, y que es necesario tenerlo prendido... me demore un ratito descubriendo que tenia que abrir la app para que no me diera error.

File: README.md [hunk 3]
```diff
@@ -29,6 +29,18 @@ Here are the steps to execute the project:
...
+   In order to generate a local database run the following statement:
+      ```
+      pnpm db:docker:startup
+      ```
+   It is required to modify `DATABASE_URL` variable at .env.* file to target the local database
+   If you want to shut down the database please execute the following statement
+      ```
+      pnpm db:docker:shutdown
+      ```
```
Comment (serudda) [inline README.md]:
```suggestion
   > It is required to modify `DATABASE_URL` variable at .env.* file to target the local database.

   If you want to shut down the database please execute the following statement
      ```
      pnpm db:docker:shutdown
      ```
```

**Review by serudda — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ksreyr) [reply .gitignore]: Seru aqui no entiendo cual seria el cambio :S

File: README.md (hunk 3 above)
Comment (ksreyr) [reply README.md]: Hecho el cambio :D

File: README.md (hunk 2 above)
Comment (ksreyr) [reply README.md]: Hecho seru :D

**Review by serudda — APPROVED** [review]

**Pull request merged**